modules/library/src/test/scala/zio/elasticsearch/QueryDSLSpec.scala

```diff
@@ -62,6 +62,11 @@ object QueryDSLSpec extends ZIOSpecDefault {
 
           assert(query)(equalTo(MatchQuery(field = "name.keyword", value = "Name")))
         },
+        test("successfully create Bool Query with boost") {
+          val query = boolQuery().boost(1.0)
```

Comment:
After new changes are applied, we are going to use `boolQuery.` instead of `boolQuery().`.